changelogs/unreleased/5976-use-pip--config-file.yml

```diff
@@ -0,0 +1,7 @@
+description: improved support for pip config by adding use_config_file to project.yml
```

Comment:
Isn't the point of release notes to briefly mention a change so that user know what to look for in the docs? I feel other descriptions aren't more descriptive than this one. Or do you think it is not clear?

Reply:
> Isn't the point of release notes to briefly mention a change so that user know what to look for in the docs?

I agree but then we should add more documentation about this change e.g. mention the fact that we can now use the config file defined in PIP_CONFIG_FILE env when setting use_config_file to true. 

> Or do you think it is not clear? 

I think mentioning the PIP_CONFIG_FILE here as well wouldn't hurt

Reply:
concerning the docs:
this is done here: https://github.com/inmanta/inmanta-core/pull/5996/files#diff-b5d17e866e545b1dc547769774bc95987546ecbbb5c7decb1035c4e6f16299acR22

which will show: https://github.com/inmanta/inmanta-core/pull/5996/files#diff-69cc23dbeac56bc1a4d681d8faf95f7da4dfc3587347c54d39c068e805b238a4R348

I'll try to add the PIP_CONFIG_FILE to the note

Reply:
> which will show: https://github.com/inmanta/inmanta-core/pull/5996/files#diff-69cc23dbeac56bc1a4d681d8faf95f7da4dfc3587347c54d39c068e805b238a4R348

Aah I completely missed the fact that this would be shown. Adding PIP_CONFIG_FILE to the note will make it easy to find :+1: